Dockerfile

```diff
@@ -37,8 +37,9 @@ WORKDIR /app
 COPY --from=builder --chown=500:500 /app /app
 
 RUN <<EOT
+    ls -la /app/bin
     useradd --system -m -d /app -U -u 500 plone
-    runDeps="git libjpeg62 libopenjp2-7 libpq5 libtiff5 libxml2 libxslt1.1 lynx netcat poppler-utils rsync wv busybox gosu libmagic1 make"
+    runDeps="git libjpeg62 libopenjp2-7 libpq5 libtiff5 libxml2 libxslt1.1 lynx netcat poppler-utils rsync wv busybox gosu libmagic1 build-essential"
```

Comment:
Why do you add build-essential here? Do we need all of it in the final image?

Reply:
Probably not need, its a leftover from my trials. I'll have a look.

Reply:
OK, I removed it.